lang/en/tool_advancedreplace.php

```diff
@@ -80,7 +80,7 @@
 $string['filespageheader'] = 'Search for text in Moodle files';
 $string['newsearch'] = 'New search';
 $string['searchdeleted'] = 'The selected search was deleted.';
-$string['searchpagename'] = 'Advanced replace search';
+$string['searchpagename'] = 'Advanced database search';
```

Comment:
Do we still need the 'advanced' at all for the page names? They are already under the heading so just file search and database search should be good.

Reply:
How do we like this:
**Advanced Search and Replace**
> Search and Replace in the Database
> Search and Replace in Files

Reply:
+1 this is good